parsl/app/bash.py

```diff
@@ -21,8 +18,20 @@ def remote_side_bash_executor(func, *args, **kwargs):
     import subprocess
     import logging
     import parsl.app.errors as pe
+    from parsl import set_file_logger
+
+    logbase = "/tmp"
```

Comment:
Would it be more polite to collect these in a dedicated directory? `/tmp/parsl_logs` or something?

Reply:
opened #1107 for something related